agent/src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -122,19 +123,20 @@ public static void fuzzerInitialize(String[] args) {
       }
       System.exit(1);
     }
-    List<Class<?>> alwaysIgnore =
-        Arrays.stream(args)
-            .skip(1)
-            .map(name -> {
-              try {
-                return ClassLoader.getSystemClassLoader().loadClass(name);
-              } catch (ClassNotFoundException e) {
-                System.err.printf("Failed to find class '%s' specified in --autofuzz_ignore", name);
-                System.exit(1);
-              }
-              throw new Error("Not reached");
-            })
-            .collect(Collectors.toList());
+
+    Set<Class<?>> alwaysIgnore = new HashSet<>();
+    ClassGraph classGraph = new ClassGraph().enableClassInfo().enableSystemJarsAndModules();
```

Comment:
Could we avoid this potentially very slow search if the arguments are not regexes (e.g. contain only `[a-zA-Z0-9.]` + Unicode)?